activerecord/test/cases/adapters/mysql2/annotate_test.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require "cases/helper"
+require "models/post"
+
+class Mysql2AnnotateTest < ActiveRecord::Mysql2TestCase
```

Comment:
Can you consolidate the duplicated `test/cases/adapters/*/annotate_test.rb`?
Those are not adapter specific tests.

Reply:
ah I see, that is for the quoting things for `assert_sql`, it is fine as is for now.